documentation/src/theme/Footer/Layout/index.js

```diff
@@ -183,25 +177,25 @@ export default function FooterLayout({
                             {socialLinks?.items?.map?.((socialLink, i) => (
                                 <a
                                     key={i}
-                                    className="w-[22px] h-[22px]"
+                                    className="w-[22px] h-[22px] hover:text-red-600"
```

Comment:
```suggestion
                                    className="w-[22px] h-[22px]"
```

We don't have text link right now and probably red will not fit our design system.
We can delete this hover style right now.